src/Renci.SshNet/.editorconfig

```diff
@@ -188,3 +188,6 @@ dotnet_diagnostic.MA0040.severity = none
 # MA0042: Do not use blocking calls in an async method
 # duplicate of CA1849
 dotnet_diagnostic.MA0042.severity = none
+
+# S3236: Caller information arguments should not be provided explicitly
+dotnet_diagnostic.S3236.severity = none
```

Comment:
Why was this one reported? Don't have time to check myself now.

Reply:
It comes when one wants to provide a value for a parameter marked with `[CallerExpressionAttribute]`, e.g.

```c#
public TimeSpan Timeout
{
    get
    {
        return _timeout;
    }
    set
    {
        // EnsureValidTimeout(this TimeSpan timeSpan, [CallerArgumentExpression(nameof(timeSpan))] string? paramName = null)

        value.EnsureValidTimeout(nameof(Timeout));

        _timeout = value;
    }
}
```

I would probably suppress this one at the call sites, but #1480 will also bring more places where we don't want it (from Debug.Assert)